website/package.json

```
@@ -49,27 +50,31 @@
     ]
   },
   "dependencies": {
-    "@docusaurus/core": "3.6.0",
-    "@docusaurus/faster": "3.6.0",
-    "@docusaurus/plugin-google-gtag": "3.6.0",
-    "@docusaurus/plugin-pwa": "3.6.0",
-    "@docusaurus/preset-classic": "3.6.0",
+    "@docusaurus/core": "^3.6.3",
```

Could these dependencies be kept pinned?
```suggestion
    "@docusaurus/core": "3.6.3",
```